website/docs/plugin/framework/migrating/resources/index.mdx

```diff
@@ -0,0 +1,225 @@
+---
+page_title: 'Resources: Migrating from SDKv2 to the Framework'
+description: >-
+  Migrate a resource from SDKv2 to the plugin Framework.
+---
+
+# Resources
+
+Resources are an abstraction that allow Terraform to manage infrastructure objects by defining create, read, update,
+and delete functionality that maps onto API operations. Resource schemas define what fields a resource has, give
+Terraform metadata about those fields, and define how the resource behaves. Refer to
+[Resources](/plugin/framework/resources) in the Framework documentation for details.
+
+This page explains how to migrate a resource's schema from SDKv2 to the plugin Framework. We also recommend reviewing these additional guides for resources throughout the migration:
+- [Create, Read, Update, and Delete functions](/plugin/framework/migrating/resources/crud): The resource defines the logic to manage resources with Terraform.
+- [Import](/plugin/framework/migrating/resources/import): The resource defines the logic to add a resource to Terraform's state.
+- [Plan modification](/plugin/framework/migrating/resources/plan-modification): The resource customizes the Terraform plan for known values or behaviors outside the practitioner's configuration.
+- [State upgrade](/plugin/framework/migrating/resources/state-upgrade): The resource updates Terraform state information in advanced use cases.
+
+## SDKv2
+In SDKv2, resources are defined by the `ResourcesMap` field in the `schema.Provider` struct, which maps resource names
+(strings) to their schema. Each schema is a `schema.Resource` struct that includes:
+
+- A `Schema` field, which defines resource attributes
+- Fields for resource lifecycle functions such as `Create` and `CreateContext`
+- Fields for functions to implement state upgrade (`StateUpgraders`), import (`Importer`), and customize diff
+(`CustomizeDiff`)
+
+The following code shows a basic implementation of resource schema with SDKv2.
+
+```go
+func New() *schema.Provider {
+    return &schema.Provider{
+        ResourcesMap:   map[string]*schema.Resource
+    }{
+        "resource_example": resourceExample(),
+    }
+    /* ... */
+},
```

Comment:
```suggestion
func New() *schema.Provider {
    return &schema.Provider{
        ResourcesMap:   map[string]*schema.Resource{
            "resource_example": resourceExample(),
        /* ... */        
    }
    /* ... */
},
```

I think the brackets and indentation were wrong, is this correct? Also added another `/* ... */` to indicate that there'd be more resources.

Reply:
Yep, brackets and indentation were incorrect. I've updated as follows:

```go
func New() *schema.Provider {
    return &schema.Provider{
        ResourcesMap:   map[string]*schema.Resource {
            "resource_example": resourceExample(),
            /* ... */
        },
        /* ... */
    }
}
```